### PR TITLE
Request endpoint user register 

### PR DESCRIPTION
**Pull request by hectorerb**
## Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: [Conventional Commit](http://conventionalcommits.org/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue # N/A

## What is the new behavior?

- Add email address to register
- Create new user

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/AdminDashboard/Api/Glpi.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import 'isomorphic-fetch'
+const URL = "https://dev.flyve.org/glpi/apirest.php/"
+// const URL = "http://glpis42.local/apirest.php/"
+const userToken = "zq6xxHdUwJfh1pnTcKi66sLvX978KKGVvOFK9LBS"
```
Comment (ajsb85) [inline src/AdminDashboard/Api/Glpi.js]: Try to use an external config file

File: src/AdminDashboard/Api/Glpi.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import 'isomorphic-fetch'
```
Comment (ajsb85) [inline src/AdminDashboard/Api/Glpi.js]: This lib was updated 3 years ago 😛

File: src/AdminDashboard/Api/Glpi.js (hunk 1 above)
Comment (hectorerb) [reply src/AdminDashboard/Api/Glpi.js]: Hi @ajsb85 
Yes, this is already designed to be used in the library

File: src/AdminDashboard/Api/Glpi.js (hunk 2 above)
Comment (hectorerb) [reply src/AdminDashboard/Api/Glpi.js]: isomorphic-fetch is implimentation of fetch for node.js and browser, built on top of fetch polyfill, but We could use https://github.com/github/fetch

Comment (ajsb85) [reply src/AdminDashboard/Api/Glpi.js]: Yes please

**Pull request merged**